adoptopenjdk-api-v3-models/src/main/kotlin/net/adoptopenjdk/api/v3/models/OperatingSystem.kt

```diff
@@ -3,13 +3,14 @@ package net.adoptopenjdk.api.v3.models
 import org.eclipse.microprofile.openapi.annotations.enums.SchemaType
 import org.eclipse.microprofile.openapi.annotations.media.Schema
 
-@Schema(type = SchemaType.STRING, enumeration = ["linux", "windows", "mac", "solaris", "aix"])
+@Schema(type = SchemaType.STRING, enumeration = ["linux", "windows", "mac", "solaris", "aix", "alpine-linux"])
 enum class OperatingSystem : FileNameMatcher {
-    linux(0, "LinuxLH", "alpine-linux"),
+    linux(0, "LinuxLH"),
     windows(0, "win"),
     mac,
     solaris,
-    aix;
+    aix,
+    `alpine-linux`;
```

Comment:
whitespace?

Reply:
the whitespace is on, note there is a backtick there